eslint/babel-eslint-parser/test/index.js

```diff
@@ -1,51 +1,85 @@
-import assert from "assert";
-import espree from "espree";
+import path from "path";
 import escope from "eslint-scope";
 import unpad from "dedent";
 import { parseForESLint } from "../src";
-import assertImplementsAST from "./helpers/assert-implements-ast";
 
-const babelOptions = {
+const BABEL_OPTIONS = {
   configFile: require.resolve(
     "@babel/eslint-shared-fixtures/config/babel.config.js",
   ),
 };
+const ALLOWED_PROPERTIES = [
+  "importKind",
+  "exportKind",
+  "variance",
```

Comment:
This is used in [`eslint-plugin-flowtype`](https://github.com/gajus/eslint-plugin-flowtype/search?q=variance&unscoped_q=variance).